pythran/analyses/cfg.py

```diff
@@ -6,6 +6,10 @@
 import networkx as nx
 
 
+#: A singleton object to use as the sink node in a graph.
+NIL = object()
+
+
 class CFG(FunctionAnalysis):
     """
```

Comment:
That's picky, but I think you should make ``NIL`` a (static) member of ``CFG``, that way you would not need to import it globally at the same level as other analysis (using ``CFG.NIL`` looks better)